src/workloads/query/TimeseriesBlockProcessing.yml

```diff
@@ -0,0 +1,356 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload runs queries on time-series collections that can at least partially run with block
+  processing. At the moment, only queries with a $match prefix are eligible for block processing.
+  The dataset used for this workload has uniformly random data so bucket level filtering is
+  ineffective. This stresses turning buckets into blocks and running block based operations on them.
+
+Keywords:
+  - timeseries
+  - aggregate
+
+GlobalDefaults:
+  Database: &database test
+  Collection: &collection Collection0
+  DocumentCount: &documentCount 1e6
```

Comment:
Can we make it 10m? A lot of time series workloads have around 100m or more so I just want to be somewhere close

Reply:
Done. Reduced number of repeats from 100 to 20 since most queries are ~10x longer.